ps_socialfollow.php

```diff
@@ -187,11 +201,44 @@ public function getConfigFieldsValues()
     {
         $result = [];
         foreach (static::SOCIAL_NETWORKS as $social) {
-            $result['blocksocial_' . $social] = Configuration::get('BLOCKSOCIAL_' . strtoupper($social));
+            $configuration_name = 'BLOCKSOCIAL_' . strtoupper($social);
+            if (!empty(Configuration::get($configuration_name))) {
+                $this->upgradeConfiguration($configuration_name);
+            }
+            foreach ($this->context->controller->getLanguages() as $lang) {
```

Comment:
I didn't find a simpler way to read the values in all the languages of a localized configuration value. I feel something built in must exist, since the translation of configuration values is natively supported in PS.

Reply:
Maybe https://github.com/PrestaShop/PrestaShop/blob/develop/classes/Configuration.php#L274 ?

I'm not expert in using Configuration class 😄  ping @kpodemski

Reply:
This method is not available as of 1.7.7.5 ...